tests/cpp_random_unit.cpp

```diff
@@ -378,7 +379,7 @@ DEFINE_TEST(random_doublecheck_test_64) {
         const Roaring64Map &right = roars[rand() % NUM_ROARS];
 
 #ifdef ROARING_CPP_RANDOM_PRINT_STATUS
-        printf("[%lu]: %llu %llu %llu\n", step,
+        printf("[%lu]: %" PRIu64 " %" PRIu64 " %" PRIu64 "\n", step,
                static_cast<unsigned long long>(left.cardinality()),
                static_cast<unsigned long long>(right.cardinality()),
                static_cast<unsigned long long>(out.cardinality()));
```

Comment:
Now that you're using `PRIu64` there should NOT be casts here any more.

Reply:
Good catch. Fixing.